Sources/Apollo/GraphQLResponse.swift

```diff
@@ -1,15 +1,26 @@
 /// Represents a GraphQL response received from a server.
-public final class GraphQLResponse<Data: GraphQLSelectionSet> {
+public final class GraphQLResponse<Data: GraphQLSelectionSet>: Parseable{
```

Comment:
You might have missed that space `Parseable{`,  `Parseable {`

Reply:
🤦‍♀️ thank you!